katello-attach-subscription

```diff
@@ -433,14 +437,14 @@ def checksubs()
         sys = @api.resource(:hosts).call(:show, {:id => system['id'], :fields => 'full'})
         if @options[:virtwho] and @virtwho_data
           if @options[:verbose]
-            puts "VERBOSE: adding virt-who data to #{sys['name'}"
+            puts "VERBOSE: adding virt-who data to #{sys['name']}"
```

Comment:
I've split out this and the next change into https://github.com/RedHatSatellite/katello-attach-subscription/pull/42 so we can get the bugfix in quicker than the whole review :)